libmscore/edit.cpp

```diff
@@ -4567,7 +4568,8 @@ void Score::undoAddElement(Element* element)
 
             int it = 0;
             for (int ntrack : tr) {
-                  if ((ntrack & ~3) != staffIdx * VOICES) {
+                  // Skip this track if it is in a linked staff within the same Score, unless adding a Tie
+                  if ((ntrack & ~3) != staffIdx * VOICES && !element->isTie()) {
```

Comment:
Same here.

Reply:
Actually, I think I can delete this bit altogether, now that I am using your method for constructing the list of tracks.